cargo-pgx/src/command/install.rs

```diff
@@ -393,7 +393,7 @@ pub(crate) fn get_version(manifest_path: impl AsRef<Path>) -> eyre::Result<Strin
                     .wrap_err("Couldn't parse manifest")?;
 
                 let version = manifest.package.ok_or(eyre!("no `[package]` section found"))?.version;
-                Ok(version.to_string())
+                Ok(version)
```

Comment:
Explicit extra allocs are things rustc/LLVM struggle to elide, so this is a good one to get.